components/31-molecules/highlight/_highlight.scss

```
@@ -35,7 +35,7 @@
   .accolade {
     &::before,
     &::after {
-      @include theme('background', 'color-primary--lighten-5', 'highlight-background');
+      @include theme('background', 'color-primary--lighten-5', 'highlight-inverted-background');
```

No need to change this one.
(it's also quite dangerous, as you'll potentially break existing themes)

I've checked and 'inverted' only exists in the contact-details, not in highlight itself.

```suggestion
      @include theme('background', 'color-primary--lighten-5', 'highlight-background');
```

---

Ah I see, I changed highlight file in place of contact details 🤦 .